spring-boot/admin/src/main/resources/templates/admin/fragments/services/edit.html

```diff
@@ -51,6 +51,12 @@ <h2 th:text="#{admin.services.edit.form.title}">
                                 <div class="form-group">
                                     <label class="control-label col-md-3 col-sm-3 col-xs-12">
                                         <span th:text="#{admin.services.edit.form.label.id}"/>
+                                        <a data-toggle="modal" data-target=".getting-started-service-identifier-modal">
+                                            <i class="fa fa-question-circle"></i>
+                                        </a>
```

Comment:
To make the message even more obvious, I would suggest to add:
```suggestion
                                        </a>
                                        <br/>
                                        <small style="color: red">Be careful when editing</small>
```